docs/auth/android.md

```diff
@@ -15,23 +15,26 @@ dependencies {
 
 ## Install the RNFirebase Authentication package
 
+**Note**: This is for react-native 0.60+ - for earlier versions of react-native please refer to the [previous version of this documentation](https://github.com/invertase/react-native-firebase-docs/blob/8c36d9df1a35e993f5d4126c33448256b9c5b682/docs/auth/android.md).
+
 Add the `RNFirebaseAuthPackage` to your `android/app/src/main/java/com/[app name]/MainApplication.java`:
 
 ```java
 // ...
-import io.invertase.firebase.RNFirebasePackage;
+import com.facebook.react.ReactApplication;
 import io.invertase.firebase.auth.RNFirebaseAuthPackage; // <-- Add this line
 
 public class MainApplication extends Application implements ReactApplication {
     // ...
 
     @Override
```

Comment:
This is still included in error? I cannot see how to override this method/

Reply:
This is intentional - this is still part of the auto generated React Native 0.6 code for MainApplication.java